### ISSUE TITLE
Feature Request: Message retention

### ISSUE THREAD

**Issue by Kimiblock** [girlbossceo/conduwuit#331]
Similar to [Synapse](https://element-hq.github.io/synapse/latest/usage/configuration/config_documentation.html?highlight=Retention#retention), implement message retention to prevent the database from growing over time.

**Comment by coomcode** [comment]
anyone have a method of doing this until it is added?

**Comment by Kimiblock** [comment]
I don’t think so. But conduwuit’s database doesn’t grow as fast as Synapse + PostgreSQL.

**Comment by Aranjedeath** [comment]
You are currently limited to clearing media; that said media is the largest share of most people's disk space usage. Check `!admin media` command in admin room for options.
